projects/plugins/jetpack/extensions/blocks/subscriptions/settings.js

```diff
@@ -3,96 +3,232 @@ import { getRedirectUrl } from '@automattic/jetpack-components';
-			return '(' + ( emailSubscribers + socialFollowers ) + '+)';
-		case accessOptions.subscribers:
-			return '(' + emailSubscribers + ')';
-		case accessOptions.paid_subscribers:
-			return '(' + paidSubscribers + ')';
+	switch ( accessOptions[ accessLevelKey ].key ) {
+		case accessOptions.everybody.key:
+			return emailSubscribers;
+		case accessOptions.subscribers.key:
+			return emailSubscribers;
+		case accessOptions.paid_subscribers.key:
+			return paidSubscribers;
 		default:
+			return 0;
 	}
 }
 
-export function MisconfigurationWarning( { accessLevel } ) {
+function NewsletterLearnMore() {
 	return (
-		<div className="jetpack-subscribe-notice-misconfiguration warning">
-			{ sprintf(
-				/* translators: %1$s: visibility label for the newsletter, %2$s: label for setting "everybody". this is a warning in the newsletter when posts have a private or password-protected visibility */
+		<small className="jetpack-newsletter-learn-more">
+			{ createInterpolateElement(
 				__(
-					'Private or password-protected posts cannot be assigned a newsletter setting of "%1$s". Please update the setting to "%2$s", or update the post visibility setting.',
+					'Restrict your post to subscribers. <learnMoreLink>Learn more.</learnMoreLink>',
 					'jetpack'
 				),
-				accessOptions[ accessLevel ].label,
-				accessOptions.everybody.label
+				{
+					learnMoreLink: (
+						<Link
+							slug={ getRedirectUrl( 'paid-newsletter-info', {
+								anchor: 'memberships-and-subscriptions',
+							} ) }
+						/>
+					),
+				}
 			) }
-		</div>
+		</small>
 	);
 }
 
-function NewsletterAccessChoices( {
-	accessLevel,
+function NewsletterNotice( { accessLevel, socialFollowers, emailSubscribers, paidSubscribers } ) {
+	// Get the reach count for the access level
+	let reachCount = getReachForAccessLevelKey(
+		accessLevel,
+		emailSubscribers,
+		paidSubscribers,
+		socialFollowers
+	);
+
+	if ( 0 === reachCount ) {
+		return (
+			<FlexBlock>
+				<Notice status="info" isDismissible={ false } className="edit-post-post-visibility__notice">
+					{ createInterpolateElement(
+						__(
+							'You don’t have any subscribers yet. How about <importingLink>importing</importingLink> some? Or check out <thisGuideLink>this guide</thisGuideLink> on how to grow your audience.',
+							'jetpack'
+						),
+						{
+							importingLink: (
+								<Link
+									slug={
+										'https://wordpress.com/support/launch-a-newsletter/import-subscribers-to-a-newsletter/'
+									}
+								/>
+							),
+							thisGuideLink: (
+								<Link slug={ 'https://wordpress.com/support/category/grow-your-audience' } />
+							),
+						}
+					) }
+				</Notice>
+			</FlexBlock>
+		);
+	}
+
+	if ( accessOptions.everybody.key === accessLevel ) {
+		reachCount = reachCount + '+'; // Concat "+"
+	}
+
+	return (
+		<FlexBlock>
+			<Notice status="info" isDismissible={ false } className="edit-post-post-visibility__notice">
+				{ __( 'This will be sent to ', 'jetpack' ) }
```

Comment:
No partial strings; they don't work for translating.

See PCYsg-3hI-p2#partial-strings-and-concatenation

Reply:
Ficed in e56f586038a32690555beee5e0c5243156ea1315